explanations/machine-learning-terms.md

```diff
@@ -70,6 +70,24 @@ Researchers often invent complicated names for simple things. This page will hel
     </ul>
 </td>
 </tr>
+<tr>
+<td valign="top">Loss (Cost) Functions</td>
+<td>
+    <p>Functions that help to evaluate your model.</p>
```

Comment:
I think this needs to be a bit more precise. The loss function is important during training and is usually not very suitable to evaluate the performance of the model.

Maybe something like: "A function that is used to evaluate the performance of a model and guide the training process."